parm/config/config.resources

```diff
@@ -249,6 +249,27 @@ elif [[ "${step}" = "ocnanalprep" ]]; then
     export npe_node_ocnanalprep
     export memory_ocnanalprep="3072M"
 
+elif [[ "${step}" = "ocnanalbmat" ]]; then
+   npes=16
+   case ${CASE} in
+      C384)
+        npes=480
+        ;;
+      C48)
+        npes=16
+        ;;
+      *)
+          echo "FATAL: Resolution not supported'"
+          exit 1
+    esac
+
+    export wtime_ocnanalbmat="00:30:00"
+    export npe_ocnanalbmat=${npes}
+    export nth_ocnanalbmat=1
+    export native_ocnanalbmat="--exclusive"
```

Comment:
Are we sure this `--exclusive` will work on all schedulers?
I think you want this line to be:
```suggestion
    export is_exclusive=True
```

Please also update the section in `ocnanalrun` where this slipped in.

Reply:
not true, a required dependency of IODA is still not available, but OOPS can compile

Reply:
Is `is_exclusive=True` a way of generalizing `sbatch --exclusive`?

Reply:
That is the intention, yes.